doc/using.rst

```diff
@@ -20,6 +20,10 @@ your Binder repository should contain at least:
 * One (or many) text files that specify the requirements of your code.
   For a complete list, see :ref:`config-files`.
 
+Configuration text files should be either in the **root** of your
+repository, or in a folder in the root of the repository that is called
+``binder``. E.g., ``myrepo/binder/requirements.txt``.
```

Comment:
Let's indent this to be part of the bullet in line 20. 
 
* Configuration files (one or more text files) that specify the requirements of building your project's code. For
   a complete list of supported files, see :ref:`config-files`. Configuration files may be placed in the **root**
   of your repository or  in a ``binder`` folder in the repository's root (i.e. ``myproject/binder/``).  If a ``binder``
   folder is used, Binder will only read configuration files from that location and will ignore those in the
   repository's root.